### ISSUE TITLE
Update DNS for demo.faq.coronavirus.gov

### ISSUE THREAD

**Issue by brandocalrissian** [18F/cv_faq#1132]
Currently managing the DNS for this on the GSA side but should be with HHS.

**Comment by brandocalrissian** [comment]
WHEREAS:

- the DNS entry for a demo federalist deployment is helpful for testing things we cannot easily test without pushing to a publicly available site,
- HHS appears to be the agency who is managing the DNS entries for faq.coronavirus.gov,
- and GSA IT is auditing DNS entries and does not want us to have a demo federalist DNS entry managed by us/GSA,
- we need to approach our friends at HHS to get a DNS entry for demo.faq.coronavirus.gov to point to our demo federalist deployment.

Why do we need this?

- It’s the only way we can test redirects before going to prod. When FAQs are deprecated, they are redirected to more current FAQs when possible using the redirect_from directive. Feature branch builds can’t be used to confirm these redirects because of the path.
- Occasionally people want to approve things before they go into production.

What needs to happen, technically?

- See the [cloud.gov documentation on setting up subdomains](https://cloud.gov/docs/services/external-domain-service/#how-to-create-an-instance-of-this-service), a more detailed version of the [federalist docs on adding subdomains](https://federalist.18f.gov/documentation/custom-domains/#adding-a-subdomain)

**Comment by brandocalrissian** [comment]
@amirbey thank you for your help! this ticket will track where we are on this.

**Comment by brandocalrissian** [comment]
Putting together a brief email to describe what we're asking for to send over to our HHS contact(s).

**Comment by brandocalrissian** [comment]
We would like to set up DNS entries for demo.faq.coronavirus.gov so we can test redirecting out-of-date FAQs to new content before pushing the changes to the production site.

Once the DNS CNAME entries are set up for _acme-challenge.demo.faq.coronavirus.gov. and demo.faq.coronavirus.gov. per the Federalist documentation (https://federalist.18f.gov/documentation/custom-domains/#adding-a-subdomain), we will work with the Federalist team to finish the configuration.

**Comment by brandocalrissian** [comment]
(catching the ticket up) email sent to HHS contact

**Comment by brandocalrissian** [comment]
Blocked on external activity... Will update when something changes...

**Comment by brandocalrissian** [comment]
We are talking more closely with HHS over the coming weeks and will bring this up again.

**Comment by brandocalrissian** [comment]
No longer needed now that the project has wrapped up.
